install.sh

```diff
@@ -1,2 +1,8 @@
-#!/bin/bash
-echo "Install"
+#!/bin/sh
+echo "Install git and clone the VOPI repository to start installation..."
+
+sudo xbps-install git
+git clone https://github.com/balintkiraly/vopi.git /tmp/vopi
+cd /tmp/vopi
```

Comment:
**[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Use 'cd ... || exit' or 'cd ... || return' in case cd fails. [SC2164](https://github.com/koalaman/shellcheck/wiki/SC2164)